packages/firebase_ml/lib/src/firebase_remote_model.dart

```diff
@@ -6,40 +6,25 @@ part of firebase_ml;
 
 /// Describes a remote model to be downloaded to the device.
 ///
-/// Defines the download conditions of the model,
-/// whether or not to download updated versions of the model,
-/// and the model's name specified by the developer in the cloud console.
-/// https://firebase.google.com/docs/reference/android/com/google/firebase/ml/common/modeldownload/FirebaseRemoteModel
-class FirebaseRemoteModel {
-  String _modelName;
-
-  /// Returns the name of your model
-  String getModelName() => this._modelName;
-}
-
-/// Describes a remote model to be downloaded to the device.
-/// Create a remote model object with the model's name specified by the developer in the cloud console.
-/// https://firebase.google.com/docs/reference/android/com/google/firebase/ml/custom/FirebaseCustomRemoteModel
-class FirebaseCustomRemoteModel extends FirebaseRemoteModel{
-
-  FirebaseCustomRemoteModel._builder(FirebaseCustomRemoteModelBuilder builder) {
-    this._modelName = builder._modelName;
-  }
+/// https://firebase.google.com/docs/reference/android/com/google/
+/// firebase/ml/common/modeldownload/FirebaseRemoteModel
+abstract class FirebaseRemoteModel {
+  /// Model's name specified by the developer in the console.
+  final String modelName;
+
+  /// Constructor for [FirebaseRemoteModel] that requires
```

Comment:
this comment provided no additional value.. document `modelName` or just remove it

Reply:
Removed the comment